code/modules/mob/living/bot/floorbot.dm

```diff
@@ -126,6 +125,11 @@
 			if(prob(1))
 				ignorelist -= T
 
+
+/mob/living/bot/floorbot/think()
```

Comment:
Lines 117 and 118 from Life() can be nuked.
Lines 123 - 126 are a part of the AI, so should fall under think, no? Might be an idea to have `stop_think` cut the list so dangling refs are cleaned and that every new AI init starts unbiased.